src/en/darthsdroids/src/eu/kanade/tachiyomi/extension/en/darthsdroids/DarthsDroids.kt

```diff
@@ -0,0 +1,222 @@
+package eu.kanade.tachiyomi.extension.en.darthsdroids
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.model.UpdateStrategy
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import rx.Observable
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+// Dear Darths & Droids creators:
+// I’m sorry if this extension causes too much traffic for your site.
+// If possible, i would use the Zip downloads to source finished
```

Comment:
Reading directly from ZIP has been done before in PandaChaika, but that was out of necessity. I would recommend against implementing that.